internal/cmd/measurement_schema/client.go

```diff
@@ -0,0 +1,284 @@
+package measurement_schema
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/influxdata/influx-cli/v2/internal"
+	"github.com/influxdata/influx-cli/v2/internal/api"
+)
+
+type Client struct {
```

Comment:
WDYT about writing unit tests for the logic in this file? It felt tedious when I did it for the `bucket` commands, but it flushed out some bugs in the process.

Reply:
I'll add some today 👍🏻

Reply:
Done.

I introduced the Go team's mock package, https://github.com/golang/mock, as we've also been using it in IDPE and it's been really useful.